packages/bpk-component-skip-link/src/BpkSkipLink.scss

```diff
@@ -0,0 +1,37 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2020 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '~bpk-mixins/index';
+
+.bpk-skip-link {
+  display: inline-block;
+  padding: $bpk-spacing-sm;
+  transition: none;
+  border-radius: $bpk-button-border-radius;
+  background-color: $bpk-color-panjin;
```

Comment:
Haven't seen the design spec but do we possibly need this to support theming? Just from a partner branding perspective so that it wouldn't clash?